### ISSUE TITLE
Calinou doesn't sleep

### ISSUE THREAD

**Issue by IntangibleMatter** [godotengine/godot-proposals#4485]
### Describe the project you are working on

Occasionally I comment on the Godot feature proposals repository. Many others also do this

### Describe the problem or limitation you are having in your project

Calinou, the person who seems to manage this repo replies to nearly every comment within 10 minutes or so. None of it seems to be automated either!

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Calinou needs to get some sleep, as he seems to... not do that.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Give Calinou some warm milk and a nice fluffy pillow and let him catch up on presumable years worth of lost rest

### If this enhancement will not be used often, can it be worked around with a few lines of script?

I don't believe human sleeping can be scripted, no.

### Is there a reason why this should be core and not an add-on in the asset library?

Because I'm concerned about the amount of sleep Calinou gets.

(please don't ban me, I just couldn't not)

**Comment by bluenote10** [comment]
Indeed, Calinou's level of professionalism is absolutely inspiring. Taking the opportunity to leave a **big thank you** for the awesome work!

**Comment by mrjustaguy** [comment]
> ### If this enhancement will not be used often, can it be worked around with a few lines of script?
> I don't believe human sleeping can be scripted, no.

This is false. Human Sleeping can be Scripted, it's just not a few Lines of Script, and requires someone that's great with Neural Networks, However why would one wish to place malicious Neural Networks that just undermine performance? There is only one thing Worth Losing Performance for (A ton of for that matter), and That's a Good DRM! Something that every Big Company/Entity Should Do With Their Products and Services! (NOT)

**Comment by MrBBBaiXue** [comment]
## A BIG THANK TO ALL GODOT CONTRIBUTORS!!!
## THANK YOU FOR PROVIDING SUCH A AWESOME GAME ENGINE!

**Comment by akien-mga** [comment]
Have you tried sending `PUT /sleep/8h` using the documented Calinou REST API?

**Comment by aaronfranke** [comment]
Another serious case of Godot addiction (Godotitis in medical terms). Has anyone found a Godoctor yet?

![image](https://user-images.githubusercontent.com/5209613/27392849-070bce28-56a8-11e7-893b-1e46cdb877e7.png)

**Comment by LinuxUserGD** [comment]
> > ### If this enhancement will not be used often, can it be worked around with a few lines of script?
> > I don't believe human sleeping can be scripted, no.
> 
> This is false. Human Sleeping can be Scripted, it's just not a few Lines of Script, and requires someone that's great with Neural Networks, However why would one wish to place malicious Neural Networks that just undermine performance? There is only one thing Worth Losing Performance for (A ton of for that matter), and That's a Good DRM! Something that every Big Company/Entity Should Do With Their Products and Services! (NOT)

Might require NFTs (for donating sleep) and https://github.com/godotengine/godot/issues/9044

**Comment by slapin** [comment]
NFTs won't do, just donate some alcohol instead

**Comment by me2beats** [comment]
can't see the issue in current alpha
It seems the "sleep mode" works now?

**Comment by shinspiegel** [comment]
This is one of the most heartwarming issues I have ever seen in this industry. 

**Thank you all for the amazing work on this project**

**Comment by shinspiegel** [comment]
> You mean in ANY industry, Right?

Probably :D

**Comment by photon-niko** [comment]
Will this be in Godot 4?

**Comment by YuriSizov** [comment]
We can only hope that Calinou would sleep after the 4.0 stable release

**Comment by autofool** [comment]
So I wasn't the only one who felt troubled by this! But this behavior is not limited to this repo. I used to ask stuff on Godot forums, reddit and dev-chat and as far as I remember Calinou is like one of the top 3 persons to reply. Is it possible that he is a ghost that's haunting everything Godot??

**Comment by jams3223** [comment]
If this keeps on, Calinou.exe is going to crash, no pun intended.

**Comment by MarkoGrbec** [comment]
> If this keeps on, Calinou.exe is going to crash, no pun intended.

Are you sure he runs on crappy windows?

**Comment by syntaxerror247** [comment]
I think we should close it as not planned.

**Comment by DaloLorn** [comment]
... But *does* he plan to sleep after we replace him with an LLM, or will he just break it?

**Comment by mrjustaguy** [comment]
Pfft, _**Transformer Models**_ are impossible to break by definition! All he can do is decide to use a bad _Projection Matrix_ that will **~bastardize~** _I mean **Transform**_ his **Model**, by improperly scaling it, rotating it, translating it or sheering it.

a very tall and thin **Bed** is still _A_ **Bed**, and so is a leaning **Bed**, or half way inside your eyes **Bed**... a sideways **Bed** too!

All of them are still very much a **Bed**....... _I think_

Likewise, **We** don't replace _`People`_ with _**Fancy Rendering technologies**_.

.....

.....

_`People`_ replace _`People`_ with _**Fancy Rendering technologies**_.
